src/dotnet/commands/dotnet-run3/Run3Command.cs

```diff
@@ -80,21 +84,16 @@ private ICommand GetRunCommand()
                 globalProperties.Add("TargetFramework", Framework);
             }
 
-            ProjectInstance projectInstance = new ProjectInstance(Project, globalProperties, null);
+            Project project = new Project(Project, globalProperties, null);
 
-            string runProgram = projectInstance.GetPropertyValue("RunCommand");
+            string runProgram = project.GetPropertyValue("RunCommand");
```

Comment:
It won't say "since it isn't built".  If you feel this is necessary, please open an issue on it.